metrics-exporter/src/bin/io_engine/collector/mod.rs

```diff
@@ -34,3 +35,19 @@ fn init_volume_gauge_vec(metric_name: &str, metric_desc: &str, descs: &mut Vec<D
     descs.extend(gauge_vec.desc().into_iter().cloned());
     gauge_vec
 }
+
+/// Initializes a GaugeVec metric for replica with the provided metric name, description and
+/// descriptors.
+fn init_replica_gauge_vec(metric_name: &str, metric_desc: &str, descs: &mut Vec<Desc>) -> GaugeVec {
+    let opts = Opts::new(metric_name, metric_desc)
+        .subsystem("replica")
+        .variable_labels(vec![
+            "node".to_string(),
+            "name".to_string(),
+            "pv_name".to_string(),
+        ]);
+    let gauge_vec = GaugeVec::new(opts, &["node", "name", "pv_name"])
+        .unwrap_or_else(|_| panic!("Unable to create gauge metric type for {}", metric_name));
```

Comment:
```suggestion
        .expect(format!("Unable to create gauge metric type for {}", metric_name));
```

Reply:
linter complains about use of expect followed by fn call. Keeping as is.